app/design/adminhtml/Magento/backend/Magento_Catalog/web/css/source/_module.less

```diff
@@ -102,3 +102,13 @@
         }
     }
 }
+
+@media screen and (min-width: 1080px) {
```

Comment:
Please use the default [Magento 2 breakpoints](https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/responsive-web-design/rwd_css.html#fedg_rwd_css_break).
The fix is not related to the `Magento_Catalog` module only. So, please add it to the appropriate LESS file.
e.g. `app/design/adminhtml/Magento/backend/web/css/source/forms/_form-wysiwyg.less`.